core/micro-services/workflow-compiling-service/src/main/scala/edu/uci/ics/texera/service/WorkflowCompilingServiceConfiguration.scala

```diff
@@ -0,0 +1,5 @@
+package edu.uci.ics.texera.service
+
+import io.dropwizard.core.Configuration
+
+class WorkflowCompilingServiceConfiguration extends Configuration {}
```

Comment:
@shengquan-ni how does this configuration relate to @DanielMao1 's project?

Reply:
Daniel's project is to separate execution config from the server config. So far it has no conflict with this change.